plugins/pulp_docker/plugins/importers/sync.py

```diff
@@ -50,6 +51,77 @@ def __init__(self, repo=None, conduit=None, config=None,
                                        working_dir, constants.IMPORTER_TYPE_ID)
         self.description = _('Syncing Docker Repository')
 
+        download_config = nectar_config.importer_config_to_nectar_config(config.flatten())
+        upstream_name = config.get(constants.CONFIG_KEY_UPSTREAM_NAME)
+        url = config.get(importer_constants.KEY_FEED)
+
+        # Create a Repository object to interact with.
+        self.index_repository = registry.V2Repository(
+            upstream_name, download_config, url, working_dir)
+        self.v1_index_repository = registry.V1Repository(upstream_name, download_config, url,
+                                                         working_dir)
+
+        v2_found = self.index_repository.api_version_check()
+        v1_found = self.v1_index_repository.api_version_check()
+        # TODO: REMOVE THIS
```

Comment:
Austin wrote:
> @rbarlow <https://github.com/rbarlow> remove this :)

my brain is running low on thinking from doing too much work. Fixed.

-- 
Randy Barlow
irc:   bowlofeggs